exercises/complex-numbers/test/Tests.hs

```diff
@@ -32,12 +34,18 @@ specs = do
           describe "sub"       $ for_ subCases       $ testA sub
  where
   testA f CaseA{..} = it descriptionA $ f (complex number1A) (complex number2A)
-                                        `shouldBe` complex expectedA
+                                        `shouldBeAround` complex expectedA
   testB f CaseB{..} = it descriptionB $ f (complex number1B)
-                                        `shouldBe` complex expectedB
+                                        `shouldBeAround` complex expectedB
   testC f CaseC{..} = it descriptionC $ f (complex number1C)
                                         `shouldBe` expectedC
+  shouldBeAround = shouldBe `on` roundComplex 2
 
+  roundComplex :: Int -> Complex Float -> Complex Float
+  roundComplex n c = complex (roundTo n (real c), roundTo n (imaginary c))
```

Comment:
I am not sure if there is a better way to do this, but we don't have access to `Complex` data constructor here so cannot do this:

```haskell
roundComplex n (Complex x y) = ..
```

Reply:
Looks good to me.